templates/ssl_ratios_v1.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-derived-datasets.telemetry.ssl_ratios_v1` AS
+WITH windowed AS (
+  SELECT
+    *,
+    SUM(ssl_loads) OVER w1 + SUM(non_ssl_loads) OVER w1 AS total_loads
+  FROM
+    `moz-fx-data-derived-datasets.telemetry.ssl_ratios_raw_v1`
+  WINDOW
+    w1 AS (ORDER BY submission_date ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING))
```

Comment:
We could optionally make this window only do preceding, so that only new values are effected when the underlying table updates:
```suggestion
    w1 AS (ORDER BY submission_date RANGE UNBOUNDED PRECEDING))
```